errors/sharp-missing-in-standalone.md

```diff
@@ -0,0 +1,27 @@
+# Missing sharp in standalone mode
```

Comment:
Can we reuse the existing error page since its nearly the same error? `/errors/sharp-missing-in-production.md`

Reply:
Yeah I think adding a note to the existing document about `output: 'standalone'` should be sufficient here.